src/routes/(app)/contacts/Graph.svelte

```diff
@@ -0,0 +1,81 @@
+<script lang="ts">
```

Comment:
This should be in `src/routes/contacts/_components` for consistency, and the file name lowercaps too